tequilapi/docs/swagger.json

```diff
@@ -4525,6 +4562,40 @@
       },
       "x-go-package": "github.com/mysteriumnetwork/node/tequilapi/contract"
     },
+    "SessionItem": {
```

Comment:
At least our PO also had ideas about `Service Type` used during the session 

![image](https://user-images.githubusercontent.com/20511486/184067866-66c107f9-56e0-4ac0-afe8-0987532d77a2.png)